static/app/views/starfish/views/spans/selectors/actionSelector.tsx

```diff
@@ -105,8 +105,8 @@ const LABEL_FOR_MODULE_NAME: {[key in ModuleName]: ReactNode} = {
   cache: t('Action'),
   vital: t('Action'),
   queue: t('Action'),
-  screen: t('Action'),
-  startup: t('Action'),
+  screen_load: t('Action'),
+  app_start: t('Action'),
```

Comment:
```suggestion
  screenLoad: t('Action'),
  appStart: t('Action'),
```
Also, maybe I'm missing context but what are these used for? 🤔